libs/core-ui/src/lib/util/JointDataset.ts

```diff
@@ -645,6 +645,34 @@ export class JointDataset {
     });
   }
 
+  public getJointDatasetFeatureName(userFeatureName: string | undefined): string | undefined {
+    // Return the joint dataset feature name for the given user feature name.
+    if (userFeatureName === undefined) {
+      return undefined;
```

Comment:
Maybe just not accept undefined then?  It seems strange to accept it in this method and return undefined.  Perhaps it might be better to just force the caller to make sure the parameter is defined before passing it in here, and handle the case with error/etc if the feature name is not defined in the code prior to this method?

Reply:
Thanks for the suggestion. Simplified the flow of the function to not take undefined value as input parameter.